### DISCUSSION TITLE
Future-ready talent self-learning problem

### DISCUSSION THREAD

**Discussion by kirangalande137** [github/india#848] [General]
Future-ready talent self-learning modules do not get updated only show in progress but I completed them already.
and i got daily mail from future ready talent to you have not compeleted any azure course

**Comment by FRTSupport** [comment]
If your self learning course 'mandatory module' status shows "in progress", please make sure that your MS learn profile URL is entered correctly in the orientation page. Also, ensure that you are logged in to your MS learn id as well as your FRT portal while doing a course, it's important for your progress to be reflected in the Badges and Trophies section of your MS learn profile.
